install.bat

```diff
@@ -42,17 +42,23 @@ IF %USE_FIFTY_ONE_DB%==true (
   echo ***** USING LOCAL MONGODB *****
 )
 
-IF %VOXEL51_INSTALL%==false (
-  echo ***** INSTALLING FIFTYONE-BRAIN *****
+echo ***** INSTALLING FIFTYONE-BRAIN *****
+IF %SOURCE_BRAIN_INSTALL%==true (
+  echo Cloning FiftyOne Brain repository
+  git clone https://github.com/voxel51/fiftyone-brain
+  cd fiftyone-brain
+  IF %DEV_INSTALL%==true (
+    ./install.bat -d
```

Comment:
this needs to be changed to include the keyword "CALL" this allows one bat script to call another and then resume once it's finished otherwise it doesn't do anything. Also, when using CALL you won't include the file path so it will look like this:
`CALL install.bat -d`
tested locally by checking out the brain branch and running the following command from fiftyone
`./install.bat -b -d`
this resulted in an output containing both install.bat outputs (here you can see the end of fiftyone-brain and then the continuation of fiftyone):
```
***** INSTALLATION COMPLETE *****
***** INSTALLING FIFTYONE *****
```

Reply:
Thanks! Makes sense